apps/methodologies/bold-carbon-organic/rule-processors/mass/project-period/README.md

```diff
@@ -0,0 +1,32 @@
+<div align="center">
+
+# Project Period
+
+Methodology: **BOLD-CARBON-ORGANIC**
+
+[![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/carrot-foundation/methodology-rules/check-and-deploy.yaml)](https://github.com/carrot-foundation/smaug/actions)
+
+</div>
+
+## 📄 Description
+
+<!-- TODO: Update README rule descriptions <https://app.clickup.com/t/3005225/CARROT-1943> -->
+
+In an event, when there is a report identified as CDF, verify if the declared addresses for the participant identified in the event and the actor identified as Recycler are the same.
+
```

Comment:
I see use have a TO-DO to update the description, so we should either remove this description or add a temporary description that actually explains what the rule does. Specially considering this is a public repository, we should avoid misconceptions.